src/com/xilinx/rapidwright/util/FileTools.java

```diff
@@ -517,10 +496,11 @@ public static void writeStringToTextFile(String text, String fileName) {
 	 * @return An ArrayList containing strings of each line in the file. 
 	 */
 	public static ArrayList<String> getLinesFromTextFile(String fileName){
-		String line = null;
+		String line;
 
 		ArrayList<String> lines = new ArrayList<String>();
-		try (BufferedReader br = new BufferedReader(new FileReader(fileName))){
+		try (FileReader fr = new FileReader(fileName);
+			 BufferedReader br = new BufferedReader(fr)) {
```

Comment:
```suggestion
			BufferedReader br = new BufferedReader(fr)) {
```
Removing stray space